modules/claims_api/app/controllers/claims_api/v1/forms/power_of_attorney_controller.rb

```diff
@@ -223,6 +224,25 @@ def build_representative_info(poa_code)
             }
           end
         end
+
+        def check_file_number_exists!
+          ssn = target_veteran.ssn
+
+          begin
+            response = bgs_service.people.find_by_ssn(ssn) # rubocop:disable Rails/DynamicFindBy
+            unless response && response[:file_nbr].present?
+              error_message = 'Unable to locate Veteran FileNumber. '\
```

Comment:
```suggestion
              error_message = 'Unable to locate Veteran file number. '\
```
assuming this messaging was discussed with Kayla and or whoever else would need to copy edit. looks fine to me other than this small tweak